sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3745,6 +3745,21 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       }
     }
   }
+
+  test("SPARK-33593: Parquet vector reader incorrect with binary partition value") {
```

Comment:
Can you add a test case in https://github.com/apache/spark/commit/f7d2143705c8c1baeed0bc62940f9dba636e705b#diff-8e4e74b1869fecc81b75575bed45b183e79cbaa97578f42c51984c9d89ece45aR694 too?

Reply:
> Can you add a test case in [f7d2143#diff-8e4e74b1869fecc81b75575bed45b183e79cbaa97578f42c51984c9d89ece45aR694](https://github.com/apache/spark/commit/f7d2143705c8c1baeed0bc62940f9dba636e705b#diff-8e4e74b1869fecc81b75575bed45b183e79cbaa97578f42c51984c9d89ece45aR694) too?

Thanks for your  advise, I am  looking where to add UT for this since not familiar with this part.